distribution/build.yaml

```diff
@@ -21,6 +21,10 @@ distribution_spec:
     eval:
     - provider_type: remote::trustyai_lmeval
       module: llama_stack_provider_lmeval==0.2.4
+    - provider_type: inline::trustyai_ragas_inline
+      module: llama_stack_provider_ragas==0.1.1
```

Comment:
this won't work until https://github.com/llamastack/llama-stack/pull/3341 merged